simulator/src/main/java/kcl/teamIndexZero/traffic/simulator/data/trafficLight/TrafficLightSet.java

```diff
@@ -10,7 +18,92 @@
 
 public class TrafficLightSet implements ISimulationAware {
     private ID id;
+    public List<TrafficLightInSet> TrafficLightSetList;
+    public List<TrafficLightInSet> InteriorListA;
+    public List<TrafficLightInSet> InteriorListB;
```

Comment:
These 3 lists: TrafficLightSetList, InteriorListA, InteriorListB - You only need 2. One for one state and one for the opposite state.